wasmbinding/wasm.go

```diff
@@ -28,3 +31,13 @@ func RegisterCustomPlugins(
 		messengerDecoratorOpt,
 	}
 }
+
+func RegisterStargateQueries(queryRouter baseapp.GRPCQueryRouter) []wasmkeeper.Option {
```

Comment:
How are you planning to propagate `StargateLayerBindings` to the logic in `wasmd`

I'm assuming the plan is to remove the `wasmd` copy of `StargateLayerBindings` and use this instead?

Reply:
We are basically adding the newly added stargatelayerbindings rather than overriding it. This is possible because the stargate whitelist is initialized empty

https://github.com/osmosis-labs/osmosis/pull/2190/files#diff-1ac782d2fab1d48ad7f7ef06cf2d58b37c99b6b4c98879a5ad86a985b73fad58R321